content/cucumber/configuration.md

```diff
@@ -22,17 +84,15 @@ want to execute with this profile.
 The example above generates two profiles:
 
 1. `html_report`, with a list of command-line options that specify new output formats, and
```

Comment:
Qu) Do we prefer to link people to what individual args do, or specify them inline? As below we stipulate what it does, but here we don't?

Reply:
@luke-hill - If you want to make changes to the Ruby parts, please do. @mpkorstanje and I have no opinion on this ;)